tasks/install_modjk.yml

```diff
@@ -0,0 +1,52 @@
+- name: Apache_httpd | Install mod_jk prerequisites
+  apt:
+    package: "{{ item }}"
+    update_cache: yes
+    cache_valid_time: 86400
+  with_items: "{{ apache_modjk_prerrequisites }}"
+
+- name: Apache_httpd | Download mod_jk
+  unarchive:
+    src: "{{ apache_modjk_url }}"
+    dest: "/tmp"
+    remote_src: yes
+
+- name: Apache_httpd | Create configure script
+  command: ./buildconf.sh
+  args:
+    chdir: /tmp/tomcat-connectors-{{ apache_modjk_version }}-src/native
```

Comment:
repeated 3 times... Would be better in a var?

Reply:
Done